aws/src/integration/java/org/apache/iceberg/aws/s3/TestFlakyS3InputStream.java

```diff
@@ -59,7 +64,8 @@ public void setupTest() {
   }
 
   @Override
-  S3InputStream newInputStream(S3Client s3Client, S3URI uri) {
+  S3InputStream newInputStream(
```

Comment:
this never returns AAL stream right? These tests are not run with AAL, I expect them to fail since AAL is not retrying today.

Reply:
Nice catch! I've missed overriding this method. Updated now and skipped those we expect to fail.